ext/standard/var_unserializer.re

```diff
@@ -653,10 +657,10 @@ static inline int object_common(UNSERIALIZE_PARAMETER, zend_long elements, zend_
 		/* Delay __unserialize() call until end of serialization. We use two slots here to
 		 * store both the object and the unserialized data array. */
 		ZVAL_DEREF(rval);
-		tmp = var_tmp_var(var_hash);
+		tmp = tmp_var(var_hash, 1);
 		ZVAL_COPY(tmp, rval);
 		Z_EXTRA_P(tmp) = VAR_UNSERIALIZE_FLAG;
-		tmp = var_tmp_var(var_hash);
+		tmp = tmp_var(var_hash, 0);
```

Comment:
In conjunction with the above comment, this can be just `tmp++` to go to the next slot.